cypress/integration/article_spec.js

```diff
@@ -0,0 +1,21 @@
+describe('News Article', () => {
+const url = 'http://localhost:3000/';
+
+    it('should display 200/OK', () => {
+        cy.request(url + 'status').then((response) => {
+            expect(response.status).to.eq(200)
+        })
+    })
+
+    it('should render a headline', () => {
+        cy.visit(url)
```

Comment:
Can this and the test below have a `beforeEach` where it runs this?

Reply:
in adding a beforeEach I'll need to seperate out the status check